pkgs/applications/kde/marble.nix

```
@@ -18,6 +18,11 @@ mkDerivation {
     qtscript qtsvg qtquickcontrols qtwebengine shared-mime-info krunner kparts
     knewstuff gpsd
   ];
+  # Remove once upstream supports gpsd 3.23.1
+  prePatch = ''
```

```suggestion
  postPatch = ''
```
If you see any other package that uses prePatch for this instead of post, please send a PR in. This is wrong and breaks applying patches.